### DISCUSSION TITLE
No answer for reporting security issue

### DISCUSSION THREAD

**Discussion by g3rzi** [git-for-windows/git#3022] [General]
Hi,

In your [website ](https://git-scm.com/community) you wrote that reporting security issues should be through:
git-security@googlegroups.com

No one is answering my mails (from 13.1.2021), can you check why?
Who is responsible for this list?

Thanks

/question

**Comment by dscho** [comment]
@g3rzi sorry for not replying yet, it is on my incredibly frustratingly large and fast-growing todo list.

**Comment by g3rzi** [comment]
@dscho no problem, I just didn't know if you received the mail. Thanks for letting me know :)

**Reply by dscho** [reply]
I finally got back to it yesterday.
